### PR TITLE
Update GSON and JClouds

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
We can move GSON forward now that we're using the newest JClouds release

#### Changes
* Update GSON version

### CODE REVIEW

**Comment by labkey-adam** [comment]
Need to force `jakarta.ws.rs:jakarta.ws.rs-api` and `jakarta.annotation:jakarta.annotation-api`?

**Review by labkey-adam — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -331,6 +335,7 @@ allprojects {
 
                     // Force consistency for dependencies from cloud
                     force "joda-time:joda-time:${jodaTimeVersion}"
+                    force "joda-time:joda-time:${jodaTimeVersion}"
```
Comment (labkey-adam) [inline build.gradle]: Oops

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Looks like the "joda-time" force line got duplicated. Maybe harmless, but probably good to get rid of it.

File: build.gradle (hunk 1 above)
Comment (labkey-jeckels) [reply build.gradle]: Fixed. I was hacking at it in the hopes of eliminating the force, but it's still needed (only one though!)

**Review by Sigmonia — APPROVED** [review]

**Comment by labkey-jeckels** [comment]
Version 3.0.0 of `jakarta.ws.rs:jakarta.ws.rs-api` repackage from `javax.`->`jakarta.`. That makes it incompatible with the usage through the Docker module. Abandoning until the Docker dependency upgrades too.

**Pull request closed without merging**